components/server/src/ome/security/SystemTypes.java

```diff
@@ -78,6 +79,8 @@ public boolean isSystemType(Class<?> klass) {
             return true;
         } else if (IGlobal.class.isAssignableFrom(klass)) {
             return true;
+        } else if (Namespace.class.isAssignableFrom(klass)) {
```

Comment:
Depending on the interpretation that's wanted, you could add `global="true"` to the mapping file and then this line wouldn't be necessary.